src/server/routes/agent.rs

```diff
@@ -12,6 +12,12 @@ use std::collections::HashMap;
 use std::sync::Arc;
 use warp::{self, Filter, Rejection};
 
+#[derive(Deserialize)]
```

Comment:
For consistency add `#[serde(rename_all = "kebab-case")]` so the `experiment_name` field is automatically renamed to `experiment-name` by serde.